browser/components/MarkdownPreview.js

```diff
@@ -1109,7 +1109,17 @@ export default class MarkdownPreview extends React.Component {
     }
 
     // other case
-    shell.openExternal(href)
+    this.openExternal(href)
+  }
+
+  openExternal (href) {
+    try {
+      const success = shell.openExternal(href) || shell.openExternal(decodeURI(href))
```

Comment:
the created Link should always be encoded and onclick decoded. Otherwise it could cause problems with edge cases when a non-encoded file looks like encoded. For example, if you have two files in the same directory:

- %E6%B8%AC%E8%A9%A6.txt
- %25E6%25B8%25AC%25E8%25A9%25A6

Reply:
As it seems the original behavior was fine under mac os, and the issue seems to be Windows-only, I made it conservative that it prioritized original behavior. If you can confirm that "always decoding" works fine under each os, I'm happy to change to that case.

Reply:
@hiiwave would you consider testing it under Windows with always decoding? I am currently testing it on Linux.

Reply:
Always decoding behaves exactly as this implementation, because it simply does not work without decoding.

Reply:
I can confirm that "always decoding" works fine under Linux.

Reply:
@hiiwave ok, I was assuming edge cases on Windows that make `shell.openExternal(href)` succeed. 

If you create a storage location in the folder `%20` both raw and decoded will fail because it must be encoded:

`shell.openExternal(encodeURI(href)`

works in this case. I don't know where `href` is stored but it must be encoded on creation.

Reply:
As https://github.com/BoostIO/Boostnote/issues/3397#issuecomment-569370829:
> IMHO, users should not create a folder named %20 in any case. I don't think anyone will ever do it as well.